src/lib/transaction_snark/test/dune

```diff
@@ -49,3 +49,4 @@
   (preprocess
     (pps ppx_snarky ppx_version ppx_jane ppx_sexp_conv))
   (instrumentation (backend bisect_ppx)))
+(ignored_subdirs (zkapp_fuzzy))
```

Comment:
Looks like `ignored_subdirs` is deprecated. Try using `data_only_dirs`? https://dune.readthedocs.io/en/stable/dune-files.html#data-only-dirs-since-1-6